crates/apollo-compiler/test_data/ok/0015_supergraph.graphql

```diff
@@ -22,6 +22,10 @@ type Amazon {
 
 union Body = Image | Text
 
+"""
+  Usually made of pages
+
+"""
```

Comment:
could we put these in a separate test_data file specifically for string serialization tests?

Reply:
Done!